packages/vue/src/components/confirm-reset-password.vue

```diff
@@ -124,15 +126,17 @@ export default defineComponent({
       getActorState(state.value)
     ) as ComputedRef<ResetPasswordState>;
     // Computed Properties
-    const backSignInText = computed(() => BACK_SIGN_IN_TEXT);
-    const lostYourCodeText = computed(() => LOST_YOUR_CODE_TEXT);
-    const resendCodeText = computed(() => RESEND_CODE_TEXT);
-    const confirmationCodeText = computed(() => CONFIRMATION_CODE_TEXT);
-    const confirmResetPasswordHeading = computed(
-      () => CONFIRM_RESET_PASSWORD_HEADING
+    const backSignInText = computed(() => I18n.get(BACK_SIGN_IN_TEXT));
```

Comment:
Should we remove the usage of `BACK_SIGN_IN_TEXT` from `DefaultTexts.ts` and instead use `Back to Sign In`? Otherwise this won't actually translate anything yet, right? 🤔 

https://github.com/aws-amplify/amplify-ui/blob/4f3dfd0df2e4bbaaff48a1dc9b12a4dd67c84433/packages/ui/src/i18n/translations.ts#L20

Reply:
@ericclemmons The default texts shows 
`export const BACK_SIGN_IN_TEXT = 'Back to Sign In';`
So that should translate correctly, since that text matches ` 'Back to Sign In': 'Zurück zur Anmeldung', `

Reply:
I did go through and update a handful of `BACK_SIGN_IN_TEXT` to match the translations files

Reply:
Oooh, I see.  So it's a bit of indirection here!  So I guess we have 2 paths in the future:

1. Directly use the `keyof translations[lang]` like `Back to Sign In` or
2. Have constants like `BACK_SIGN_IN_TEXT` (https://github.com/aws-amplify/amplify-js/blob/46ee5dd91c61f49bad4da8286b2f97c737d96631/packages/amplify-ui-components/src/common/Translations.ts#L4) who's value matches something in `translations`.

I think we're good here, but we'll need to revisit I18n in a way that creates a better contract for ourselves & customers to customize:
> https://docs.amplify.aws/ui/customization/translations/q/framework/react/